test/s3select_test.cpp

```diff
@@ -1352,7 +1392,27 @@ TEST(TestS3selectFunctions, inoperatorfalse)
         true   // aggregate call
         ); 
     ASSERT_EQ(status, 0); 
-    ASSERT_NE(s3select_result, std::string("true,\n"));
+    ASSERT_EQ(s3select_result, std::string("true,\n"));
+}
+
+TEST(TestS3selectFunctions, inoperatormore)
+{
+    s3select s3select_syntax;
+    const std::string input_query = "select \"true\" from stdin where \"a\" in (\"b\", \"a\", \"d\", \"e\", \"f\");" ;
+    auto status = s3select_syntax.parse_query(input_query.c_str());
```

Comment:
goot test for /IN/
please add the mix types (and not just constant , also use expressions) , to make sure operator can handle that.

something like (handle different type comparison) 
where 10 in ( 5.0*2.0 , 12+1 , 9+1.2)